[debug] [2025-05-24T11:47:26.447Z] ----------------------------------------------------------------------
[debug] [2025-05-24T11:47:26.449Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\zuley\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js login
[debug] [2025-05-24T11:47:26.449Z] CLI Version:   14.4.0
[debug] [2025-05-24T11:47:26.449Z] Platform:      win32
[debug] [2025-05-24T11:47:26.450Z] Node Version:  v20.18.3
[debug] [2025-05-24T11:47:26.450Z] Time:          Sat May 24 2025 14:47:26 GMT+0300 (GMT+03:00)
[debug] [2025-05-24T11:47:26.450Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-05-24T11:47:26.451Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2025-05-24T11:47:26.976Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2025-05-24T11:47:26.976Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
[debug] [2025-05-24T11:48:58.642Z] ExitPromptError: User force closed the prompt with SIGINT
    at Interface.sigint (C:\Users\zuley\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\@inquirer\core\dist\commonjs\lib\create-prompt.js:101:37)
    at Interface.emit (node:events:518:28)
    at Interface.emit (node:domain:489:12)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1125:18)
    at ReadStream.onkeypress (node:internal/readline/interface:264:20)
    at ReadStream.emit (node:events:530:35)
    at ReadStream.emit (node:domain:489:12)
    at emitKeys (node:internal/readline/utils:371:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (node:internal/readline/emitKeypressEvents:64:36)
[error] 
[error] Error: An unexpected error has occurred.
